### ISSUE TITLE
Installation dependencies?

### ISSUE THREAD

**Issue by pierrepo** [betatim/notebook-as-pdf#10]
This is more a question than an issue on itself :smile: 

The installation procedure for `notebook-as-pdf` is quite straightforward:
```
python -m pip install notebook-as-pdf
pyppeteer-install
```

I had a quick look at the files inside the `binder` repo that are used to build the demo with mybinder:

- `notebook-as-pdf` is installed through `requirements.txt`
- and the `postBuild` script runs `pyppeteer-install`

However, the `apt.txt` file contains a list of Ubuntu/Debian libraries. Are they related to the installation of `notebook-as-pdf` on itself?

**Comment by betatim** [comment]
Yes, I needed to install the extra packages on the Ubuntu base image that is used by repo2docker. However I haven't spent much time check if this is really the minimal set of extra packages to install or not. If you find something that isn't strictly necessary it would be nice to know/remove it.

**Comment by pierrepo** [comment]
A quick test told me `apt.txt` is required, at least some of its content.
